### PR TITLE
alchemical analysis recipe added

### PR DESCRIPTION
**Pull request by tsenapathi**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/alchemical-analysis/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+conda config --add channels omnia
```
[the remark this answers is not in the document]
Comment (tsenapathi) [reply recipes/alchemical-analysis/build.sh]: @bgruening How can add the dependencies installed from omnia channel?

[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/alchemical-analysis/build.sh]: You can't you need to add the dependency to conda-forge as well.

**Comment by tsenapathi** [comment]
@bgruening This works with LINUX and OSX systems

**Comment by tsenapathi** [comment]
@bgruening now using the conda-forge pymbar package

**Pull request merged**